keras_rcnn/layers/object_detection/_anchor_target.py

```diff
@@ -268,12 +269,12 @@ def unmap(data, count, inds_inside, fill=0):
     size count) """
 
     if keras.backend.ndim(data) == 1:
-        ret = keras.backend.ones((count,), dtype=keras.backend.floatx()) * fill
+        ret = tensorflow.ones((count,), dtype=keras.backend.floatx()) * fill
 
         inds_nd = keras.backend.expand_dims(inds_inside)
     else:
-        ret = (count,) + keras.backend.int_shape(data)[1:]
-        ret = keras.backend.ones(ret, dtype=keras.backend.floatx()) * fill
+        ret = (count, keras.backend.shape(data)[1])
+        ret = tensorflow.ones(ret, dtype=keras.backend.floatx()) * fill
```

Comment:
Is there an issue with `keras.backend.ones`?

Reply:
```tensorflow.ones``` is somewhat different than ```keras.backend.ones```

Reply:
I'm not sure why they're different but ```keras.backend.ones``` can't handle when I try to define a tensor with shape given by ```keras.backend.shape```